### PR TITLE
from six.moves import xrange for Python 3

### PR DESCRIPTION
**Pull request by cclauss**

### CODE REVIEW

File: extras/qpsclient.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ def start_requests(self):
 
         slots = int(self.slots)
         if slots > 1:
-            urls = [url.replace('localhost', '127.0.0.%d' % (x + 1)) for x in xrange(slots)]
+            urls = [url.replace('localhost', '127.0.0.%d' % (x + 1)) for x in range(slots)]
```
Comment (voith) [inline extras/qpsclient.py]: or may be `from six.moves import xrange` to get the generator version in both versions.

Comment (cclauss) [inline extras/qpsclient.py]: I thought about that but slots should never be too large.

**Comment by cclauss** [comment]
from six.moves import xrange

File: extras/qpsclient.py (hunk 1 above)
Comment (redapple) [inline extras/qpsclient.py]: I think it's fine here to use `range()` for both Python 2 and Python 3

**Comment by cclauss** [comment]
Either this PR or #2849

**Comment by dangra** [comment]
closed on favour of #2849

**Pull request closed without merging**